Assets/__Scripts/UI/PersistentUI.cs

```diff
@@ -93,10 +93,20 @@ private void Awake()
         CMInputCallbackInstaller.PersistentObject(transform);
         LocalizationSettings.SelectedLocale = Locale.CreateLocale(Settings.Instance.Language);
         AudioListener.volume = Settings.Instance.Volume;
+
+        UpdateDSPBufferSize();
+        
         centerDisplay.host = this;
         bottomDisplay.host = this;
     }
 
+    private static void UpdateDSPBufferSize()
+    {
+        var config = AudioSettings.GetConfiguration();
+        config.dspBufferSize = (int) Math.Pow(2, Convert.ToInt32(Settings.Instance.DSPBufferSize));
```

Comment:
Is there any reason why you are using `Convert.ToInt32()` when the variable you're trying to convert is... already an integer?

Reply:
You saw nothing